### DISCUSSION TITLE
设置媒体目录和下载目录时只有一级目录

### DISCUSSION THREAD

**Discussion by Leiserx** [jxxghp/MoviePilot#2618] [问题咨询]
试了docker内安装和win的安装，但是去设置媒体保存目录和下载目录都只有一个 "/"根目录，没办法选择其他目录。
![d17a7e62cffddffd5e6c624542a210b7](https://github.com/user-attachments/assets/93666354-3d19-42ad-96cd-3efb847500a2)

**Comment by Leiserx** [comment]
文件管理目录下是可以看到所有的目录的。
![image](https://github.com/user-attachments/assets/63a11e3e-de06-44bd-b287-841afc7b2411)

**Comment by mooccat** [comment]
我也是

**Comment by zzzh2023** [comment]
你点一下/前面那个文件图标就出来了，鼠标移过去点击它

**Reply by mooccat** [reply]
谢谢，可以了

**Reply by Leiserx** [reply]
感谢，是我犯傻了，没想到图标能点。
